main.c

```diff
@@ -968,180 +1370,6 @@ void HandlePngToFullwidthJapaneseFontCommand(char *inputPath, char *outputPath,
     FreeImage(&image);
 }
 
-void HandleLZCompressCommand(char *inputPath, char *outputPath, int argc, char **argv)
```

Comment:
honestly I'd rather these remain where they are, be marked static, and have a declaration at the top of the file
I'll make a proper review later on, but this just stood out to me

Reply:
Should this be done for every applicable function or only those relevant to this PR?

Reply:
sorry for late response, it might be a bit too much work to do all of them, so keep it only those relevant to the PR for now